
From: Sam Ravnborg <sam@ravnborg.org>

When using a separate output directory the in-kernel config wiere rebuild
each time the kernel was compiled.  Fix this by specifying correct path to
Makefile in the prerequisite to the ikconfig.h file.

Signed-off-by: Sam Ravnborg <sam@ravnborg.org>
Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/kernel/Makefile |    2 +-
 1 files changed, 1 insertion(+), 1 deletion(-)

diff -puN kernel/Makefile~avoid-rebuild-of-ikcfg-when-using-o= kernel/Makefile
--- 25/kernel/Makefile~avoid-rebuild-of-ikcfg-when-using-o=	2004-06-20 14:06:48.898616576 -0700
+++ 25-akpm/kernel/Makefile	2004-06-20 14:06:48.903615816 -0700
@@ -49,7 +49,7 @@ quiet_cmd_ikconfig = IKCFG   $@
       cmd_ikconfig = $(CONFIG_SHELL) $< .config $(srctree)/Makefile > $@
 
 targets += ikconfig.h
-$(obj)/ikconfig.h: scripts/mkconfigs .config Makefile FORCE
+$(obj)/ikconfig.h: scripts/mkconfigs .config $(srctree)/Makefile FORCE
 	$(call if_changed,ikconfig)
 
 # config_data.h contains the same information as ikconfig.h but gzipped.
_
